app/Observers/ProjectObserver.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Observers;
+
+use App\Models\Project;
+use App\Models\Stat;
+
+class ProjectObserver
+{
+
+    public $afterCommit = true;
+
+    /**
+     * Handle the Project "created" event.
+     *
+     * @param  \App\Models\Project  $project
+     * @return void
+     */
+    public function created(Project $project)
+    {
+        // TASK: on creating a new project, create an Observer event to run SQL
+        //   update stats set projects_count = projects_count + 1
+        print("salut");
+        Stat::increment('projects_count', 1);
```

Comment:
I have confusion here.
Stat::increment('projects_count', 1); is working fine for me also but
why we are not fetching the first Stat Model then incrementing the projects_count?

Reply:
Calling `increment` on the model without specifying which row will increment that column for **ALL** rows, the methods `increment` and `decrement` returns the number of affected rows.

This works great here since the `Stat` table will probably only ever have one row in this example.

If you want to specify which row to update though use `first`, `find`, `where`, or any other method for selecting the correct row.

Hope that clears up the confusion. 👍